### PR TITLE
Update locality records

### PR DESCRIPTION
**Pull request by stepps00**
This PR adds polygons to the localities of Gyumri and Step'anavan. The existing point geometries were stored as alt files, and existing label centroids were kept as-is.

All of the locality points that fall within the new locality polygons look to be neighbourhoods, so they were superseded into new neighbourhood records (rather than flagged as funky). This PR needs to be PIP'd once approved.

**Number of records updated**: 14

### CODE REVIEW

**Comment by stepps00** [comment]
Screenshot 2023-06-28 at 3 29 56 PM
Screenshot 2023-06-28 at 3 30 15 PM

File: data/186/399/105/7/1863991057.geojson [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        }
+    ],
+    "wof:id":1863991057,
+    "wof:lastmodified":1687990135,
+    "wof:name":"Shahumyan S.",
+    "wof:parent_id":1108783129,
+    "wof:placetype":"neighbourhood",
+    "wof:repo":"whosonfirst-data-admin-am",
+    "wof:superseded_by":[],
+    "wof:supersedes":[],
```
Comment (nvkelso) [inline data/186/399/105/7/1863991057.geojson:71]: Please point back at the locality record this neighbourhood supersedes

**Review by nvkelso — CHANGES_REQUESTED** [review]
Same superseding work needed as the other PR, please.

**Review by nvkelso — APPROVED** [review]
Thanks!

**Pull request merged**